content/1_General/IO.mdx

```diff
@@ -495,6 +520,24 @@ int main() {
 
 Again, `BufferedReader` and `PrintWriter` should be used. The provided constructor `public Kattio(String problemName)` handles file input / output for you.
```

Comment:
I think the sentence "The provided constructor `public Kattio(String problemName)` handles file input / output for you." can just be deleted. Kattio is explained below already.